ci/docker/runtime_functions.sh

```diff
@@ -178,6 +225,8 @@ build_centos7_cpu() {
 build_centos7_mkldnn() {
     set -ex
     cd /work/mxnet
+    export CC="ccache gcc"
+    export CXX="ccache g++"
```

Comment:
How come you don't use the wrapper here?

Reply:
At first I have tried it and unfortunately the build breaks due to make build of dependencies (mshadow). This was a necessary revert to Pedros solution.